main.go

```diff
@@ -12,36 +14,70 @@ import (
 // JSON example
 // {"ar": ["joaothallis"], "elixir": ["josevalim", "eksperimental"]}
 
-func main() {
-	dir := os.Args[1]
-	homeDir, err := os.UserHomeDir()
+func getPrarFilePath() (string, error) {
+	globalFlag := flag.Bool("global", false, "When you choose to using ./config/prar.json")
+	flag.Parse()
+	if *globalFlag {
+		homeDir, err := os.UserHomeDir()
+		return homeDir + "/.config/prar.json", err
+	} else {
+		homeDir, err := os.Getwd()
+		return homeDir + "/.prar.json", err
```

Comment:
You can use `./` to get from the current dir
```suggestion
		return "./.prar.json", err
```